### ISSUE TITLE
KFA import/export 

### ISSUE THREAD

**Issue by DigitalBox98** [niftools/blender_niftools_addon#556]
**Is your feature request related to a problem? Please describe.**
KFA anim format is not supported whereas it's a NIF-like format (DAOC)

**Describe the solution you'd like**
Propose the import/export for .kfa files

**Describe alternatives you've considered**
None

**Additional context**
I've been able to create a quick & dirty import for KFA (DAOC) to animate the corresponding NIF, however I'm wondering if the menu should be included as an additional item or in the .kf menu (.kf and .kfa are both for animations)

<img width="209" alt="menu" src="https://user-images.githubusercontent.com/57635141/214429860-bb59f0ca-1721-4a81-9009-5cf435786bb2.png">

Below is what is working today (import only) : 

<img width="1403" alt="kfa" src="https://user-images.githubusercontent.com/57635141/214429948-2288a803-741e-4227-80c7-f3e86b52864e.png">

**Comment by Candoran2** [comment]
Is kfa just another name for .kf? Or does it need updates on other places too, besides the UI?

**Comment by DigitalBox98** [comment]
From my understanding it’s an older or specific kf-like format meaning kf is not compatible with kfa and vice-versa

**Comment by Candoran2** [comment]
I see. In that case, if changes need to be made to the actual import/export code you can make the changes yourself and open a PR, or at the very least post some kfa files. Either way it helps to have kfa files available for testing and verification, but if you make the code changes yourself it's obviously much quicker (especially if you've already got it working).

**Comment by DigitalBox98** [comment]
Yes my idea is to propose a PR but I was wondering if I need to separate KFA versus KF menu (DAOC is using kfa for animation not kf files)

**Comment by Candoran2** [comment]
I guess that really depends on how similar they are, but to judge that I'd need to see one.

**Comment by DigitalBox98** [comment]
Ok understood. I will propose a first PR version with a KFA/NIF in that case :)

**Comment by Candoran2** [comment]
GitHub usually lets you attach a zip file via simple drag and drop.

**Comment by DigitalBox98** [comment]
For testing purpose only : 
- dr_rat_run.kfa : first kind of kfa (ExtraData array)
- hulk_run.kfa : second kind of kfa (ExtraData chained list)

[nif_kfa.zip](https://github.com/niftools/blender_niftools_addon/files/10513941/nif_kfa.zip)
